client/src/components/App/Content/LandingPage/adminLandingPage/useAdminLandingPage.ts

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import logger from 'logger/logger';
+import { Severity } from 'models/Logger';
+import { landingPageRoute } from 'Utils/Routes/Routes';
+import FilterRulesVariables from 'models/FilterRulesVariables';
+import InvesitgationStatistics from 'models/InvestigationStatistics';
+interface Parameters {
+    setIsLoading: React.Dispatch<React.SetStateAction<boolean>>;
+    setInvestigationsStatistics: React.Dispatch<React.SetStateAction<InvesitgationStatistics>>;
+    investigationInfoFilter: FilterRulesVariables;
+}
+
+const useAdminLandingPage = (parameters: Parameters) => {
+
+    const { setIsLoading, setInvestigationsStatistics, investigationInfoFilter } = parameters;
+    
+    const history = useHistory();
+    
+    useEffect(() => {
+        const unallocatedCountLogger = logger.setup('query investigation statistics');
```

Comment:
You can move the fetch logic outside of the hook and keep the hook set state logic in the useEffect.